service/controller/chart/resources.go

```diff
@@ -135,9 +151,15 @@ func newChartResources(config chartResourcesConfig) ([]resource.Interface, error
 	}
 
 	resources := []resource.Interface{
+		// tiller migration deletes tiller once helm 3 migration is done.
 		tillerMigrationResource,
+		// namespace creates the release namespace and allows setting metadata.
 		namespaceResource,
+		// release max history ensures not too many helm release secrets are created.
+		releaseMaxHistoryResource,
+		// release manages Helm releases and is the most important resource.
 		releaseResource,
+		// status resource manages the chart CR status.
```

Comment:
We have these comments in other operators and I think are useful here too. 🤞 we can delete tiller migration soon.